pkg/sql/conn_executor.go

```diff
@@ -2457,6 +2457,7 @@ func (ex *connExecutor) execCopyIn(
 		ex.planner.maybeLogStatement(
 			ctx,
 			ex.executorType,
+			true, /* isCopy */
 			int(ex.state.mu.autoRetryCounter),
```

Comment:
completely unrelated, but i just realized we are accessing `mu` here, but didn't take the lock. maybe it's fine since nothing else can really happen during COPY

Reply:
mmmm i think we do the same thing in the "normal path"